libs/ui/ui-shared/src/lib/components/CustomModal/CustomModal.tsx

```diff
@@ -0,0 +1,33 @@
+import { Modal } from 'antd';
+
+type CustomModalProps = {
+  children: React.ReactNode;
+  title: string;
+  open?: boolean;
+  onOk?: () => void;
+  onCancel?: () => void;
+};
+
+export const CustomModal = ({
+  children,
+  title,
+  open,
+  onOk,
+  onCancel,
+  ...props
+}: CustomModalProps) => {
+  return (
+    <Modal
+      {...props}
+      title={title}
+      closable={{ 'aria-label': 'Custom Close Button' }}
```

Comment:
I do not see such API in the antd docs 
![image](https://github.com/user-attachments/assets/213bb3e9-710a-4a0d-b57d-183b456b89e2)
https://ant.design/components/modal